certora/specs/GuardianStorage.spec

```diff
@@ -370,6 +370,7 @@ rule addGuardianChangesEntries {
     address toAdd;
```

Comment:
**As a follow up**, can we merge the FV specs?

Reply:
With this PR, we are now running FV on `SocialRecoveryModuleHarness` contract. So, I don't think we need to merge the specs.
https://github.com/5afe/CandideWalletContracts/blob/78bb5d28578f19f41b767f6e03e180624572d6ac/certora/conf/GuardianStorage.conf#L19